src/components/HomePageContent/HowItWorksSection/Steps.tsx

```diff
@@ -4,8 +4,8 @@ import { StepDetail, StepsProps, ContentType } from "./types";
 const Step: FunctionComponent<StepDetail> = (props) => {
   const { icon, title, description } = props;
   return (
-    <div className={`flex flex-row flex-none w-6/12 mt-8 justify-center lg:w-auto ${icon ? "" : "lg:ml-6"}`}>
-      <div className="flex flex-col max-w-[11.5rem] min-w-[11.5rem] justify-center">
+    <div className={`flex flex-row flex-none w-full xs:w-1/2 lg:w-auto mt-8 justify-center  ${icon ? "" : "lg:ml-8"}`}>
+      <div className="flex flex-col max-w-[11.5rem] min-w-[10rem] justify-center">
         {icon && (
```

Comment:
there's no need for these changes. remove negative margins of `lg:-mx-6` and `lg:-mx-8` should do it.

Reply:
ok understand, but the issue is with mobile screens, so i will remove the ```lg:-mx-6``` as i will need the ```lg:-mx-8``` for the dotted lines, if not it will look very far away.